glusterfs/map.jinja

```diff
@@ -18,7 +18,14 @@
         'pkgs': ['glusterfs-fuse'],
         'mount_defaults': 'defaults',
     },
-}, merge=salt['pillar.get']('glusterfs:client')) %}
+}, merge=salt['grains.filter_by']({
+    'trusty': {
+        'mount_defaults': 'defaults,nobootwait'
+    },
+    'xenial': {
+        'mount_defaults': 'defaults,nofail'
```

Comment:
I don't think `nofail` should be used as default option for xenial. Also it's using systemd so mounting sequence works in a different way than using `mountall`

Reply:
So you mean good way is keep only "defaults" (without nofail and nobootwait, because this option isn't supported in ubuntu 16.04 ) ?

Reply:
Yes, I would make a condition based on init system, if it's systemd, then don't use nobootwait, etc.